README.md

```diff
@@ -12,14 +12,14 @@
     <!-- <img src="" alt="Logo" width="80" height="80"> -->
   </a>
 
-  <h3 align="center">HOMER</h3>
+  <h2 align="center">HOMER</h2>
 
   <p align="center">
     HOMER is an Open Source smart home emulator.
     <br />
```

Comment:
This whole part right here (the centered badges) feels a bit unnecessary, but LGTM

Reply:
> This whole part right here (the centered badges) feels a bit unnecessary, but LGTM

We can move them to the bottom or completely delete them, this is a template.

Reply:
i'd rather delete them, but that' s just my opinion.

Reply:
Okie dokie. I'll remove them and then squash and merge ok?

Reply:
ok